### ISSUE TITLE
Problem loading games - error

### ISSUE THREAD

**Issue by jaw9170** [TASVideos/BizHawk#2747]
Hello,

When using the most recent GIT version of BizHawkm whenever I load a game I get the following:

This happens with any ROM I try to load.  This was working at one point.  Where am I going wrong?

System.NotSupportedException: The given path's format is not supported.
   at System.Security.Permissions.FileIOPermission.EmulateFileIOPermissionChecks(String fullPath)
   at System.Security.Permissions.FileIOPermission.QuickDemand(FileIOPermissionAccess access, String fullPath, Boolean checkForDuplicates, Boolean needFullPath)
   at System.IO.FileInfo.Init(String fileName, Boolean checkHost)
   at BizHawk.Common.PathExtensions.PathExtensions.MakeAbsolute(String path, String cwd)
   at BizHawk.Client.EmuHawk.MainForm..ctor(Config config, IGL gl, Action`1 updateGlobalSound, String[] args, IMovieSession& movieSession)
   at BizHawk.Client.EmuHawk.Program.SubMain(String[] args)

Thank you,
Jason

**Comment by YoshiRulz** [comment]
What's the command you're running? Double-check which characters need to be escaped, it's likely that the shell is eating backslashes.

**Comment by jaw9170** [comment]
Thank you Yoshi,

Here is an example of the command being run:

EmuHawk.exe "D:\emu\snes\Champions World Class Soccer.sfc"

All of my SNES roms are in d:\emu\snes directory
Trying to run game named Champions World Class Soccer.sfc

If I remove the quotes EmuHawk will run but will nut load the ROM.

Any help will be appreciated.

Jason

**Comment by YoshiRulz** [comment]
So this is from the Command Prompt? And the rom can be loaded from within EmuHawk? I also keep my roms on a different drive to BizHawk but I haven't seen this error before. Try opening the Command Prompt in `D:\emu\snes` and running `C:\wherever\EmuHawk.exe "Champions World Class Soccer.sfc"`.

**Comment by jaw9170** [comment]
Yes, this is from command prompt.
The ROM can be loaded directly from within EmuHawk, yes

I tried a different game and tried what you suggested.  The game loaded fine both in command line and within EmuHawk

Command used:

D:\Emu\GEN>d:\emuhawk\emuhawk.exe "championship Bowling.7z" 

game loads fine

If I go back to EmuHawk's directory and try to run the game I get an error:

D:\EmuHawk>emuhawk.exe "d:\emu\gen\championship bowling.gen"

error:

System.NotSupportedException: The given path's format is not supported.
   at System.Security.Permissions.FileIOPermission.EmulateFileIOPermissionChecks(String fullPath)
   at System.Security.Permissions.FileIOPermission.QuickDemand(FileIOPermissionAccess access, String fullPath, Boolean checkForDuplicates, Boolean needFullPath)
   at System.IO.FileInfo.Init(String fileName, Boolean checkHost)
   at BizHawk.Common.PathExtensions.PathExtensions.MakeAbsolute(String path, String cwd)
   at BizHawk.Client.EmuHawk.MainForm..ctor(Config config, IGL gl, Action`1 updateGlobalSound, String[] args, IMovieSession& movieSession)
   at BizHawk.Client.EmuHawk.Program.SubMain(String[] args)

**Comment by YoshiRulz** [comment]
> Has the loading of ROMs changed?

The method `MakeAbsolute` listed in that stacktrace is new, it was added in 7a7522f28.

Unfortunately I wasn't able to reproduce the error. I tried these:
```
F:\BizHawk\BizHawk-2.6.1>.\EmuHawk.exe F:\BizHawk\Conker.n64
F:\BizHawk\BizHawk-2.6.1>.\EmuHawk.exe C:\users\groom\Downloads\Conker.n64
F:\BizHawk\BizHawk-2.6.1>.\EmuHawk.exe \\192.168.0.16\public\roms\N64\Conker.n64
```
...and all worked.

**Comment by jaw9170** [comment]
Well, that settles that - if you cannot reproduce the error, then the issue is on my end.

I do appreciate your time and effort.

Have a great night!

**Comment by jaw9170** [comment]
To further confirm that the issue is indeed on my end, I deleted EmuHawk and deleted all registries entries (had to do this).   I downloaded a fresh EmuHawk and now everything is working.

Thank you again and sorry for the red herring.

Jason

**Comment by anewuser** [comment]
I have the same problem after overwriting BizHawk 2.6.1 with 2.6.2.
 
 @jaw9170 What is the path to the registry directory that you've deleted?

**Comment by jaw9170** [comment]
I deleted all instances of EmuHawk.  Not sure what one(s) is causing the issue.  After deleting all registry instances of EmuHawk, I restarted EmuHawk and it works fine now.

**Comment by zeromus** [comment]
emuhawk doesn't do anything in the registry. I too want to know what registry keys you found for emuhawk. Maybe you security software marked some executables or directories as banned somehow.

**Comment by jaw9170** [comment]
well, it was working.  I just updated to 2.6.2 and now I'm getting those errors again.  Something changed in EmuHawk that's causing these exceptions.

**Comment by jaw9170** [comment]
Reopening issues as multiple users are reporting it.

**Comment by zeromus** [comment]
Pay attention to what youre deleting in the registry this time when you delete registry data to fix it

**Comment by jaw9170** [comment]
It's not with the registry.  Not sure what the deal was last time.  To be honest, I tried so many things I forget half of what I did.  To err on the side of caution, let's leave the registry alone for now.

**Comment by Nicholas-Steel** [comment]
I can confirm this is fixed after updating from 2.6.2 to 2.7.0 (I also had the issue).
